ci/tests/fputil.bash

```diff
@@ -22,24 +23,16 @@ function fputil_action {
     export WORKDIR="${DEPLOYMENT}/${3}"
```

Comment:
You are never passing in `${3}` so this could be:

```
export WORKDIR="${DEPLOYMENT}
```

Reply:
Incorporated.